ui/src/main/java/edu/wpi/grip/ui/DeployController.java

```diff
@@ -160,7 +162,8 @@ private void deploy(String address, String user, String password, String javaHom
             setStatusAsync("Running GRIP", false);
             Session session = ssh.startSession();
             session.allocateDefaultPTY();
-            Session.Command cmd = session.exec(javaHome + "/bin/java -jar " + deployDir + "/" + GRIP_JAR + " " + deployDir + "/" + PROJECT_FILE);
+            Session.Command cmd = session.exec(javaHome + "/bin/java -jar " + deployDir + "/" + GRIP_JAR + "' '"
+                    + deployDir + "/" + PROJECT_FILE + "'");
```

Comment:
Shouldn't we support uploading multiple project files and allowing the user to pick which one that they want to use?